src/services/SettingsService.ts

```diff
@@ -30,3 +38,33 @@ export const update = async (
 
   return apiService.post(endpoint, renamedSettings)
 }
+
+export const getPassword = async ({
+  siteName,
+}: {
+  siteName: string
+}): Promise<BackendPasswordSettings> => {
+  const endpoint = getSettingsPasswordEndpoint(siteName)
+  return apiService
+    .get<BackendPasswordSettings>(endpoint)
+    .then((res) => res.data)
```

Comment:
[ts error]: Parameter 'res' implicitly has an 'any' type.

Reply:
I'm not getting this on my end - the type should be specified by `apiService
    .get<BackendPasswordSettings>`

Reply:
ya can ignore this sorry error in my local machine